web/src/app/components/bar-chart/bar-chart.component.ts

```diff
@@ -229,6 +228,11 @@ export class BarChartComponent implements AfterViewInit {
       .attr('transform', `translate(${marginFocus.left}, 0)`)
       .style('color', BarChartComponent.COLOR_LIGHT_GRAY);
 
+    this.focus.append('text')
+      .attr('text-anchor', 'middle')
+      .attr('transform', `translate(${elementWidth / 2}, ${marginFocus.top / 2})`)
+      .style('font-size', '16px sans-serif')
+      .text('Bar chart of all pushes');
```

Comment:
It should be 'Bar chart of completed pushes' if a user select to show just completed pushes.

One way is to change the title based on the selection, another easier way is to just call it 'Bar chart for push durations'